scripts/select_appointments.sql

```diff
@@ -1 +1,18 @@
 -- retrieve appointment information
+SELECT p.given_name, d.surname, a.appointment_time
+FROM patients p
+INNER JOIN doctors d
+  ON d.doctor_id = d.id
+INNER JOIN appointments a
+  ON a.appointment_id = a.id
+WHERE d.surname = 'Hubbard'
+;
+
+SELECT p.given_name, d.surname, a.appointment_time
+FROM patients p
+INNER JOIN doctors d
+  ON d.doctor_id = d.id
+INNER JOIN appointments a
+  ON a.appointment_id = a.id
+WHERE p.given_name = 'Marsha Wilcox'
```

Comment:
"Marsha Wilcox" is a full name. "Marsha" is the given name, and "Wilcox" is the surname.